### PR TITLE
v1.21 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+ - Fixed a few mobile app crash / fatal error issues.
+ - Fixed an issue where having the sidebar open at all times on tablets did not work for split view.
+ - Fixed an issue on Android where channel sorting didn't match the web app.
+ - Fixed an issue where sharing a GIF via keyboard resulted in an error screen.
+ - Fixed an issue where long-press menu could not be dragged up when rotating the device to landscape view while the menu was open.
+ - Fixed an issue on Android where push notification settings were only saved after closing the settings page.
+ - Fixed an issue where users on View Members list had an icon that appeared to be selectable but was not.
+ - Fixed an issue where "Jump To" showed archived channels the user did not belong to instead of the ones the user was a member of.
+ - Fixed an issue where changing the timezone setting manually to "Set automatically" did not work on the mobile app.
+ - Fixed an issue where setting a position field for AD/LDAP sync in the System Console did not block the user from changing it in account settings.
```
Comment (enahum) [inline CHANGELOG.md]: AD/LDAP sync or SAML

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+ - Fixed an issue where **Channel Info > Manage/View Members** screen didn't load channel users.
+ - Fixed an issue where enabling large fonts on iOS caused the left-hand side text to be cut off.
+ - Fixed an issue on Android where users could not reply to a push notification if the mention was in a thread message.
+
+### Known Issues
+  - Buttons inside ephemeral posts are not clickable / functional on the mobile app. [MM-15084](https://mattermost.atlassian.net/browse/MM-15084)
+  - App slows down when opening a channel with large number of animated emoji. [MM-15792](https://mattermost.atlassian.net/browse/MM-15792)
```
Comment (enahum) [inline CHANGELOG.md]: this seems to be Android only

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.21.0 Release
+- Release Date: July 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
```
Comment (enahum) [inline CHANGELOG.md]: should we bump the minimum server version to the new ESR?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+
+### Combatibility
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
```
Comment (enahum) [inline CHANGELOG.md]: if the above is true, this is no longer needed

**Comment by amyblais** [comment]
@enahum / @hmhealey Can you help merge this PR?

**Comment by enahum** [comment]
@amyblais it has conflicts, you need to fix those

**Comment by amyblais** [comment]
@enahum Done

**Comment by amyblais** [comment]
This is just a changelog, does it need to be cherry picked?

**Comment by enahum** [comment]
@amyblais not really, removed the label.. will merge once @esethna approves

**Comment by amyblais** [comment]
@enahum Ready to merge

**Pull request merged**